src/components/HomeCard/index.js

```diff
@@ -0,0 +1,259 @@
+import React, { useState } from "react";
+import Slider from 'react-input-slider';
+import style from "./styles";
+
+
+const HomeCard = ({ className }) => {
+    const [valor, setState] = useState({ x: 0 });
+    let titles = ["No que podemos te ajudar?", "De quantos seguidores você precisa?", "De quantas curtidas você precisa?", "De quantas curtidas e seguidores você precisa?"];
+    let followers = ["1000", "1500", "3000", "6000"]
+    let likes = ["25", "50", "100", "200"]
+    let followersprices = ["55,90", "75,90", "145,90", "279,90"]
+    let likesprices = ["25,90", "45,90", "84,90", "159,90"]
+    let combosprices = ["89,90", "125,90", "249,90", "489,90"]
+    let linksfollower = ["http://mon.net.br/17s8xk", "http://mon.net.br/17s8xl", "http://mon.net.br/17s8xm", "http://mon.net.br/17s8xn"]
+    let linkslikes = ["http://mon.net.br/17s8xv", "http://mon.net.br/17s8xx", "http://mon.net.br/17s8y0", "http://mon.net.br/17s8y1"]
+    let linkscombos = ["http://mon.net.br/17s8y3", "http://mon.net.br/17s8y4", "http://mon.net.br/17s8y5", "http://mon.net.br/17s8y7"]
+    let [actualTitle, setTitle] = useState(titles[0]);
+    return (
+        <>
+            <div className={className}>
+                <div className={`card mb-9 mx-auto ${actualTitle === titles[0] ? "" : "card-increase"}`}>
+                    <div className="bg-blue block top-0 rounded-t-lg w-full h-20">
+                        <h2 className={`text-white px-4 text-center ${actualTitle === titles[0] ? "pt-6" : "pt-3 lg:pt-6"}`}>{actualTitle}</h2>
+                    </div>
+                    <div className={`flex flex-col items-center mt-12 ${actualTitle !== titles[0] ? "lg:mt-1" : ""}`}>
+                        {actualTitle === titles[0] &&
+                            <>
+                                <button onClick={() => setTitle(titles[1])} className="btn text-white mb-5">Seguidores</button>
+                                <button onClick={() => setTitle(titles[2])} className="btn text-white mb-5">Curtidas</button>
+                                <button onClick={() => setTitle(titles[3])} className="btn text-white mb-5">Seguidores <br></br>e Curtidas</button>
+                            </>}
+
+                        {actualTitle === titles[1] &&
+                            <>
+                                <div className="lg:grid lg:grid-cols-2 lg:items-center">
```

Comment:
A lot of code is being repeated here. Only a small part changes in each case of the conditional rendering after the first. A refactor should be made in order to avoid such repetition. You could for example test if the "actualTitle" is different from title[0], and if that's the case the base block of code for the card is rendered, with another conditional render just for the small part that changes.